client/tests/acceptance/land-use-action-fields-validate-test.js

```diff
@@ -53,24 +53,41 @@ module('Acceptance | land use action fields validate', function(hooks) {
       pasForm: this.server.create('pas-form'),
     });
 
+    // representation of input that is 1 over the maximum character limit
+    const maximum = (maxLength, singleCharacter) => singleCharacter.repeat(maxLength) + singleCharacter;
```

Comment:
Looking at how you're using this, it seems like the parameters `singleCharacter` is giving you the flexibility of either `string` of `number` inputs.  This might be more declarative in its usage if the interface was, for example:

```js
await fillIn('[data-test-input="dcpZoningauthorizationpursuantto"]', maximum(250, String));
// or for number
await fillIn('[data-test-input="dcpZoningauthorizationpursuantto"]', maximum(250, Number));
```

Reply:
How are you imagining defining `String` and `Number`?

Reply:
Was thinking something like this? 
```js
    const maximum = (maxLength, inputType) => {
       if (inputType === String) {
         'a'.repeat(maxLength + 1) 
        } else if ( inputType === Number ) { 
           '5'.repeat(maxLength + 1)
        }
    }
```

Reply:
Okay looks good, will make some changes :)